nifi-nar-bundles/nifi-iceberg-bundle/nifi-iceberg-processors/src/test/java/org/apache/nifi/processors/iceberg/TestPutIcebergWithHiveCatalog.java

```diff
@@ -134,12 +133,12 @@ private void initCatalog(PartitionSpec spec, String fileFormat) throws Initializ
         runner.setProperty(PutIceberg.CATALOG, "catalog-service");
     }
 
-    @ParameterizedTest
-    @ValueSource(strings = {"avro"})
-    public void onTriggerPartitioned(String fileFormat) throws Exception {
+    @Test
+    public void onTriggerPartitioned() throws Exception {
         PartitionSpec spec = PartitionSpec.builderFor(USER_SCHEMA)
                 .bucket("department", 3)
                 .build();
+        final String fileFormat = "avro";
```

Comment:
Please use `org.apache.iceberg.FileFormat` instead of strings. Also you can pass this parameter directly to the `initCatalog` method since it is only used there.

Reply:
Yes the parameter type can be changed in the `initCatalog` method and you can call `.name()` in it.